packages/gallery/src/components/gallery/blueprintsIndex.js

```diff
@@ -16,8 +16,22 @@ export default class BaseGallery extends React.Component {
     let galleryProps = { ...otherProps, styles: _styles, eventsListener: _eventsListener, domId};
 
 
-    if(this.props.useBlueprints) {
-      //
+    if (this.props.useBlueprints) {
+      try {
+        const {layoutFixer} = window;
+        if (typeof window !== 'undefined' && layoutFixer && !layoutFixer.hydrated) {
+          galleryProps.structure = layoutFixer.structure;
+          galleryProps.container = layoutFixer.container;
+          galleryProps.items = layoutFixer.items;
```

Comment:
if this is not redundant we have a problem in hydrate production.
items and styles should be the same as in SSR. container and structure are the "fixed" props.

Reply:
We get the items in the worker anyway. and it should be the exact same amount of items that we get in the hydrate